test/gtest/common/test.cc

```diff
@@ -19,6 +19,7 @@ unsigned test_base::m_total_warnings = 0;
 unsigned test_base::m_total_errors   = 0;
 std::vector<std::string> test_base::m_errors;
 std::vector<std::string> test_base::m_warnings;
+std::vector<std::string> test_base::m_fatal_warnings;
```

Comment:
m_fatal_warnings means only the first warnings or errors, right?
if so, maybe m_first_warns_and_errors?
'fatal' has a bit more serious meaning imho.

Reply:
'fatal' since these are causing the test to fail (unlike m_errors / m_warnings which are wrapped/hidden)

Reply:
'first' implies that too. also, those are not only warnings, but errors too.
i think it would be more accurate. you're call.

Reply:
@alinask renamed